x-pack/plugins/apm/public/components/app/transaction_details/transaction_details_tabs.tsx

```diff
@@ -81,6 +81,15 @@ export function TransactionDetailsTabs() {
     });
   };
 
+  // When filtering in either the latency correlations or failed transactions correlations tab,
+  // scroll to the top of the page and switch to the 'Trace samples' tab to trigger a refresh.
+  const onFilter = () => {
```

Comment:
nit: this can be called in a callback hook

Reply:
Fixed in d5bf6b55181.